pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -296,14 +296,25 @@ func (c *Controller) startSyncController(tc *corev1b1.FederatedTypeConfig) error
 	// cluster-scoped KubeFed control plane.  A namespace-scoped
 	// control plane would still have to use a non-shared informer due
 	// to it not being possible to limit its scope.
+
+	ftc := tc.DeepCopyObject().(*corev1b1.FederatedTypeConfig)
 	kind := tc.Spec.FederatedType.Kind
```

Comment:
Consider replacing `tc` with the `ftc` copy since we've moved the copy before this line.